[2022-12-16T23:12:41.331+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: titanic_etl_pipeline.create_dashboard_task manual__2022-12-16T23:11:35.228323+00:00 [queued]>
[2022-12-16T23:12:41.352+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: titanic_etl_pipeline.create_dashboard_task manual__2022-12-16T23:11:35.228323+00:00 [queued]>
[2022-12-16T23:12:41.353+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-16T23:12:41.360+0000] {taskinstance.py:1284} INFO - Starting attempt 2 of 3
[2022-12-16T23:12:41.361+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-16T23:12:41.390+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): create_dashboard_task> on 2022-12-16 23:11:35.228323+00:00
[2022-12-16T23:12:41.397+0000] {standard_task_runner.py:55} INFO - Started process 1560 to run task
[2022-12-16T23:12:41.403+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'titanic_etl_pipeline', 'create_dashboard_task', 'manual__2022-12-16T23:11:35.228323+00:00', '--job-id', '94', '--raw', '--subdir', 'DAGS_FOLDER/titanic_etl.py', '--cfg-path', '/tmp/tmpo68qz189']
[2022-12-16T23:12:41.407+0000] {standard_task_runner.py:83} INFO - Job 94: Subtask create_dashboard_task
[2022-12-16T23:12:41.532+0000] {task_command.py:389} INFO - Running <TaskInstance: titanic_etl_pipeline.create_dashboard_task manual__2022-12-16T23:11:35.228323+00:00 [running]> on host 14c90533c456
[2022-12-16T23:12:41.672+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=titanic_etl_pipeline
AIRFLOW_CTX_TASK_ID=create_dashboard_task
AIRFLOW_CTX_EXECUTION_DATE=2022-12-16T23:11:35.228323+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-16T23:11:35.228323+00:00
[2022-12-16T23:12:41.721+0000] {logging_mixin.py:137} INFO - Dash is running on http://127.0.0.1:8050/
[2022-12-16T23:12:41.721+0000] {dash.py:1968} INFO - Dash is running on http://127.0.0.1:8050/

[2022-12-16T23:12:41.745+0000] {logging_mixin.py:137} INFO -  * Serving Flask app '***'
[2022-12-16T23:12:41.747+0000] {logging_mixin.py:137} INFO -  * Debug mode: off
[2022-12-16T23:12:41.748+0000] {logging_mixin.py:137} WARNING - Address already in use
[2022-12-16T23:12:41.749+0000] {logging_mixin.py:137} WARNING - Port 8050 is in use by another program. Either identify and stop that program, or start the server with a different port.
[2022-12-16T23:12:41.776+0000] {local_task_job.py:159} INFO - Task exited with return code 1
[2022-12-16T23:12:41.917+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
